content/blog/2016/2016-05-25-gsoc-jenkins-web-ui-project.adoc

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "GSoC Project: New Job Creation and Configuration"
+- jenkins
+- core
+- jenkins2
+- gsoc
+author: samatdav
+---
+
+=== Abstract
+Although powerful, Jenkins new job creation and configuration process may be non obvious and time consuming. This can be improved by making UI more intuitive, concise, and functional. I plan to achieve this by creating a simpler new job creation, configuration process focused on essential elements, and embedding new functionality. 
+
+=== Deliverables
```

Comment:
Would be great to add some explanatory text and maybe remove some low-important bullets. It would allow to improve the list readability.